.gitignore

```diff
@@ -9,5 +9,7 @@ dist/*
 doc/source/examples/.ipynb_checkpoints/*
 **.asv
 *.orig
-MATLAB/+qc/personalPaths.mat
-/doc/source/_autosummary/*
+.idea/
+.mypy_cache/*
+.gitignore
+tests/hardware/WX2184C.exe
```

Comment:
Please do not remove `MATLAB/+qc/personalPaths.mat`.
Why do you ignore `.gitignore`?
Please add a newline at the end.

Reply:
I have ignored the gitignore with me, so that e.g. the WX2184C.exe does not stand with in the gitignore of everyone.
But since it is the gitignore in the git, ignoring it makes no sense anymore, so i removed it from the gitignore.
I also added MATLAB/+qc/personalPaths.mat and /doc/source/_autosummary/* again.